libethereum/Client.cpp

```diff
@@ -615,11 +621,14 @@ void Client::rejigSealing()
 
             if (wouldSeal())
             {
-                sealEngine()->onSealGenerated([=](bytes const& header){
-                    if (!this->submitSealed(header))
+                sealEngine()->onSealGenerated([=](bytes const& header) {
+                    LOG(m_logger) << "Block seal generated";
```

Comment:
`"Block sealed"`?

Reply:
Well right after this message there goes (on trace verbosity) "Sealing block!" https://github.com/ethereum/cpp-ethereum/blob/dacd74b4d0bba7ef131b19726e4346662f3dd4de/libethereum/Block.cpp#L840

So now it's like seal generated, then this seal is appended to the block.

Reply:
Yes, I've seen also "Generating seal..." which sounds better than "Sealing..."

Reply:
Yeah, I think I'll just delete "Sealing block!" which is confusing